### DISCUSSION TITLE
Is it possible to show two table columns in one single column?

### DISCUSSION THREAD

**Discussion by crohit7** [Budibase/budibase#6365] [Help]
I want to show title and description column together in one single column. so is this possible?
I want to achieve this
![image](https://user-images.githubusercontent.com/37010744/174285199-4eb1ebab-9361-4543-bec7-aabddfce4ff4.png)

so far I am able to achieve this
![image](https://user-images.githubusercontent.com/37010744/174285345-d410117c-d94f-447c-85f3-427aede40d3c.png)

please help me

**Comment by melohagan** [comment]
Hey @crohit7 

It is possible through Formulas: https://docs.budibase.com/docs/formula

Here's a similar discussion: https://github.com/Budibase/budibase/discussions/5213

**Reply by crohit7** [reply]
@melohagan thanks for the solution,
But this solution gives me Title and Description in the same line rather I want it to be on different lines

**Reply by melohagan** [reply]
It doesn't seem possible within a table, but if you have a Data Provider with Repeater, then you have a lot more flexibility over formatting in general.
For example using a data provider/repeater combo, you could achieve something much closer to your first screenshot using Cards (https://docs.budibase.com/docs/card)

If you use a Card Block, that will combine the Data Provider, Repeater, and Card components into a single component for convenience. 
Then the title can be the title, and subtitle the description. 
Or you can use Containers instead (https://docs.budibase.com/docs/container) to have full control over the appearance of each "row"
